host_vars/dalite.harvardx.harvard.edu/public.yml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Also this variables file is pretty generic, shared by all hosts - can we use the values here for all hosts, and simply override things specific to some hosts in the private config?